### DISCUSSION TITLE
Nfdump and the filed ethernet type

### DISCUSSION THREAD

**Discussion by Becheikh** [phaag/nfdump#561] [Q&A]
Hi Folks, 
 
Is it possible to get an ethernet type using nfdump when I try to use as a netflow analyzer?

best regards

**Comment by phaag** [comment]
You would need a netflow exporter, which exports that field with ID 256. Up to now, nfdump does not collect that field, as there was no demand so far.

**Comment by Becheikh** [comment]
Thanks for your answer. I'm interested in that field

**Comment by phaag** [comment]
Do you have an exporter sending this tag? If so send me a sample pcap to my email in the authors file.
